docs/docs/integrations/frameworks/helidon.md

```diff
@@ -0,0 +1,22 @@
+---
+sidebar_position: 9
+---
+
+# Helidon
+
+[Helidon](https://helidon.io/) provides a LangChain4j integration module that simplifies to build AI-driven applications while leveraging Helidon’s programming model and style.
+
+Helidon’s LangChain4j integration provides following advantages instead of using LangChain4j library directly:
+
+- Integration with Helidon Inject
+    - Automatically creates and registers selected LangChain4j components in the Helidon service registry based on configuration.
+- Convention Over Configuration 
+    - Simplifies configuration by offering sensible defaults, reducing manual setup for common use cases.
+- Declarative AI Services
+    - Supports LangChain4j’s AI Services within the declarative programming model, allowing for clean, easy-to-manage code structures.
+- Integration with CDI
+    - Thanks to the Helidon Inject to CDI Bridge, LangChain4j components can be used in CDI environments, including Helidon MP applications.
```

Comment:
Using the Helidon Inject to CDI bridge, LangChain4j components can be used in CDI environments, such as Helidon MP (MicroProfile) applications.

Reply:
Fixed.